apps/docs/components/popover/popover.css

```diff
@@ -1,16 +1,13 @@
-.hd-popover__dialog {
+.hd-popover {
     --hd-popover-shadow: 0 0.25rem 0.625rem 0.25rem rgb(60 60 60 / 8%);
 
     background-color: var(--hd-color-neutral-surface);
-    border: var(--hd-border-size) solid var(--hd-color-neutral-text-weak);
+    border: var(--hd-border-size) solid var(--hd-color-neutral-border);
     border-radius: var(--hd-border-radius-md);
-    box-sizing: border-box;
     box-shadow: var(--hd-popover-shadow);
-    outline: none;
-}
-
-.hd-popover:focus-visible {
-    outline: 0;
-    box-shadow: var(--hd-focus-ring);
-    border-radius: var(--hd-border-radius-sm);
+    overflow-y: scroll;
```

Comment:
oh should have put the other comment here, but again, I don't think we should be forcing a scrollbar.

Reply:
On mac at least its not visible except if it needs to have a scrollbar